test/bedpe_test.py

```diff
@@ -48,3 +48,23 @@ def test_clodius_aggregate_bedpe():
 
     cdt.get_tileset_info(output_file)
     # TODO: Make assertion
+
+
+def test_clodius_aggregate_bedpe2():
+    '''Use galGal6 chromsizes file'''
+    input_file = op.join(testdir, 'sample_data', 'galGal6.bed')
+    chromsizes_file = op.join(testdir, 'sample_data', 'galGal6.chrom.sizes')
+
+    output_file = '/tmp/blah.bed2ddb'
```

Comment:
It's a little bit tidier to use the tempfile utils:
```
with tempfile.TemporaryDirectory() as tmpdirname:
...
```
- The outputs could potentially be large, and it's nice to clean up.
- If we get around to testing the output 😄, it would be better to be sure we're starting with a clean slate; If we use the same path every time, we might just be seeing the results of a previous run.

Reply:
Done!